README.md

```diff
@@ -2,6 +2,10 @@
    <img src="https://travis-ci.org/innolitics/dicom-standard.svg?branch=master">
 </a>
 
+<a href="https://github.com/russellkan/dicom-standard/actions?query=workflow%3Aupdate-standard">
+   <img src="https://github.com/russellkan/dicom-standard/workflows/update-standard/badge.svg">
```

Comment:
Nice touch and attention to detail!

Reply:
A little ironic, but I just realized this actually links to my fork rather than this repository. I'll fix it!